packages/core/src/build/service.ts

```diff
@@ -31,6 +34,9 @@ export type Service = {
   // static
   common: Common;
   srcRegex: RegExp;
+  serverRegex: RegExp;
```

Comment:
I think it might be better to consistently use "api" instead of "server" when referring to API functions.

Also after reading this file, I wonder if it would be easier to have an `indexingRegex` and an `apiRegex`. For now, the `indexingRegex` would be everything within `src/` excluding `src/api/`.